### DISCUSSION TITLE
Marketing SVGs

### DISCUSSION THREAD

**Discussion by Jkense** [tailwindlabs/tailwindcss#14638] [Help]
Hi all, 

I am beginning to see more and more of these illustrations on websites, and now also in the TailwindUI Bento Grid. I am beginning to suspect that there is a library providing these SVGs/components. Is that indeed the case and where can I find that?

![image](https://github.com/user-attachments/assets/a0ad5754-1fa4-472f-9e0d-ee2d4be24926)

Thank you!

**Comment by adamwathan** [comment]
Hey! We designed all these ourselves I’m afraid, but we would love to find the time to create a resource full of these at some point, maybe in the new year 🤞

**Reply by Jkense** [reply]
Would be a good idea. I might just give it a shot myself. Thank you!
